src/History.tsx

```diff
@@ -64,7 +64,12 @@ class HistoryContainer extends React.Component<Props, {}> {
                 sendPostBack(this.props.botConnection, value, this.props.user, this.props.format.locale);
                 break;
 
+            case "call":
             case "openUrl":
+            case "playAudio":
+            case "playVideo":
+            case "showImage":
+            case "downloadFile":
             case "signin":
                 window.open(value);
```

Comment:
Could be true for the mime types that don't resolve to ```text/*``` 
But for ```case "openUrl":``` and ```case "signin":``` these might display in the current window if they are html